src/EnergyPlus/DataSizing.hh

```diff
@@ -65,17 +66,34 @@ struct EnergyPlusData;
 
 namespace DataSizing {
 
+    enum class OAFlowCalcMethod
     // parameters for outside air flow method
-    constexpr int NumOAFlowMethods(9);
-    constexpr int OAFlowNone(0);
-    constexpr int OAFlowPPer(1);
-    constexpr int OAFlow(2);
-    constexpr int OAFlowPerArea(3);
-    constexpr int OAFlowACH(4);
-    constexpr int OAFlowSum(5);
-    constexpr int OAFlowMax(6);
+    {
+        Invalid = -1,
+        PerPerson,    // set the outdoor air flow rate based on number of people in the zone
+        PerZone,      // sum the outdoor air flow rate per zone based on user input
+        PerArea,      // sum the outdoor air flow rate based on zone area
+        ACH,          // sum the outdoor air flow rate based on number of air changes for the zone
+        Sum,          // sum the outdoor air flow rate of the people component and the space floor area component
+        Max,          // use the maximum of the outdoor air flow rate of the people component and the space floor area component
+        IAQProcedure, // Use ASHRAE Standard 62.1-2007 IAQP to calculate the zone level outdoor air flow rates
+        PCOccSch,     // ProportionalControlBasedOnOccupancySchedule, Use ASHRAE Standard 62.1-2004 or Trane Engineer's newsletter (volume 34-5) to
+                      // calculate the zone level outdoor air flow rates based on scheduled occupancy
+        PCDesOcc,     // ProportionalControlBasedOnDesignOccupancy, Use ASHRAE Standard 62.1-2004 or Trane Engineer's newsletter (volume 34-5) to
+                      // calculate the zone level outdoor air flow rates based on design occupancy
+        Num
+    };
 
-    extern Array1D_string const cOAFlowMethodTypes;
+    constexpr std::array<std::string_view, static_cast<int>(OAFlowCalcMethod::Num)> OAFlowCalcMethodNames{
```

Comment:
Better to put this in the `.cc` file. If it is needed in more than one module, leave an `extern const` declaration here.

Reply:
Please address this also.